src/fsharp/lex.fsl

```diff
@@ -1077,7 +1087,11 @@ and verbatimString sargs skip = parse
  | "{"
     { let (buf, fin, m, kind, args) = sargs 
       if kind.IsInterpolated then
-          args.interpolatedStringNesting <- args.interpolatedStringNesting + 1
+          match args.interpolatedStringNestingCurrent, args.interpolatedStringNestingStack with
+          // initial state, depth = 0
+          | 0, _ -> ()
+          | x, y -> args.interpolatedStringNestingStack <- x :: y
```

Comment:
Style-wise, this should probably have a new line since the `-> ... <-` syntax is a little jarring on the same line

Reply:
-> LOCKED ON <-   :p
fixed.